simulators/ethereum/eels/execute-blobs/Dockerfile

```
@@ -17,8 +17,8 @@ RUN git clone --depth 1 https://github.com/ethereum/execution-specs.git && \
         git checkout FETCH_HEAD; \
     fi
 
-WORKDIR /execution-specs/packages/testing
-RUN uv sync
+WORKDIR /execution-specs/
+RUN uv sync --all-extras
```

Do not merge. I think we can remove this. Just need to check.
```suggestion
RUN uv sync
```